### ISSUE TITLE
First start dialogue: Account creation process does not start

### ISSUE THREAD

**Issue by flogat** [shell-fire/shellfire-vpn#37]
On first start of Shellfire VPN for Windows, if I select "Yes" to create a new user account in the 2nd dialogue, nothing happens.

**Comment by Tcheutchoua-Steve** [comment]
Yes, I think it's related to https://github.com/shell-fire/shellfire-vpn/issues/20#issuecomment-705824727. I will update

**Comment by flogat** [comment]
@mbehrsf please retest

**Comment by Tcheutchoua-Steve** [comment]
I'm not sure this is solved yet. I started the implementation but went on with other simpler issues like #31

**Comment by mbehrsf** [comment]
issue is still present in the current build

**Comment by flogat** [comment]
implemented with #40, ready for retest in new build

**Comment by flogat** [comment]
@mbehrsf please provide your test results here and close if tests are successful.

**Comment by mbehrsf** [comment]
issue is still present in the current build

**Comment by flogat** [comment]
Works for me, please provide details.

To simulate a fresh installation, please delete your ShellfireVPN.properties file in 
C:\users\username\AppData\Roaming\ShellfireVPN

**Comment by mbehrsf** [comment]
This whole folder gets successfully deleted when I uninstall the app, so nothing to delete there. I tried uninstalling and re-installing, but still nothing happens when I click register on first start or "Keine Zugangsdaten" afterwards.

**Comment by flogat** [comment]
Strange - the functionality is implemented and works fine for me.

Please do the following:
- Start the app
- Click on "Keine Zugangsdaten"
- Then nothing happens on your end
- Then please login, enter showconsole and submit a log

Thanks

**Comment by mbehrsf** [comment]
done, log report received at 14:01h

**Comment by flogat** [comment]
Thanks, I think I found it. New build provided - please retrieve from the usual url and let me know how it goes...

**Comment by mbehrsf** [comment]
looks good, closing :+1:

**Comment by mbehrsf** [comment]
Unfortunately, the issue is back in the latest build.

**Comment by flogat** [comment]
Please describe the exact issue.

**Comment by mbehrsf** [comment]
Same as before, nothing happens when I want to register an account on first start or when I click the registration button later on.

**Comment by flogat** [comment]
well noted, new build available - please test again, it is fixed for me.

**Comment by mbehrsf** [comment]
nop, still there. Sent log at 11:51h.

**Comment by flogat** [comment]
Sorry, please try again. Make sure to really get the newest version when you download, e.g. by using an anonymous browser tab. In Chrome, I received cached versions a few times when I tried it.
